mlrun/common/schemas/object.py

```diff
@@ -71,6 +71,7 @@ class Config:
 
 class ObjectKind(mlrun.common.types.StrEnum):
     project = "project"
+    artifact = "artifact"
```

Comment:
it annoys me that some start with capital letter and some with lowercase letter.
can it be changed to "Artifact"? with the hope that one day we'll change the other ones as well